contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -2081,9 +2081,9 @@ SELECT out_object_id as object_id
   , out_graph_type_desc as graph_type_desc
   , cast(tsql_get_expr(d.adbin, d.adrelid) AS sys.nvarchar(4000)) AS definition
   , 1::sys.bit AS uses_database_collation
-  , 0::sys.bit AS is_persisted
+  , 1::sys.bit AS is_persisted
```

Comment:
babelfish is always of type persisted for generated columns
refer first paragraph of https://www.postgresql.org/docs/current/ddl-generated-columns.html

> PostgreSQL currently implements only stored generated columns.